src/autoscaler/mta.tpl.yaml

```diff
@@ -13,7 +13,7 @@ modules:
       GO_INSTALL_PACKAGE_SPEC: code.cloudfoundry.org/app-autoscaler/src/autoscaler/metricsforwarder/cmd/metricsforwarder
       DT_RELEASE_BUILD_VERSION: ${mta-version}
     requires:
-    - name: config
+    - name: metricsforwarder-config
     - name: policydb
```

Comment:
metricsforwader also depends on bindingdb, is it included in policydb user provided service?